qemu/tests/virtio_net_dpdk.py

```diff
@@ -157,6 +157,21 @@ def install_moongen(session, ip, user, port, password, dpdk_bind_cmd):
             if session.cmd_status(cmd_bind) != 0:
                 test.error("Fail to bind nic %s on monngen host" % i)
 
+    def unbind_dpdk_nic(session, ip, user, port, password, dpdk_bind_cmd):
+        """
+
+        Clean the evn on Moongen host
+
```

Comment:
Docstring for the parameters please.

Reply:
I have update. please help review , thanks